### PR TITLE
External linking for address and phone number, store card info icon, and misc products screen styling

### PR DESCRIPTION
**Pull request by thumn**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
- Store address and phone number are linkable in `StoreDetailsScreen`
- Replaced the "See Products" button with an info icon that redirects to the products screen for that store
- Store card redirects to the map screen
- Styled product screen, including adding a header indicating number of products and time of last delivery
- Get Directions button on `StoreDetailsScreen`

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### Online sources
[Flatlist](https://reactnative.dev/docs/flatlist)
[`ItemSeparatorComponent` for Flatlists](https://stackoverflow.com/questions/43831211/itemseparatorcomponent-call-back-for-each-row)

[//]: # "Optional - copy links to any tutorial or documentation that was useful to you when working on this PR"

### Related PRs
#96 
#98 

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review
`ProductsScreen` should be rendering normally on different screen sizes. Let me know if truncation gets clunky with store name titles in the store card with the new info icon.

[//]: # "The order in which to review files and what to expect when testing locally"

## Next steps
Bug squashing :')

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases
- Accounted for edge case where phone number is not provided
- Basically tested with a few stores, mainly Holiday Market which has 21 products

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"

### Screenshots
IMG_2716
IMG_2717
IMG_2718
IMG_2719

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @anniero98 @wangannie

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: components/store/StoreCard.js [hunk 1]
```diff
@@ -57,44 +51,37 @@ export default function StoreCard({
...
-                    snapOrEbtAccepted,
-                    seeProduct
-                  ),
-                }}
-                numberOfLines={1}
-                ellipsizeMode="tail">
-                {storeName}
-              </Title>
-            )}
+            <Title
```
Comment (wangannie) [inline components/store/StoreCard.js:61]: This should still convert to Subhead on `StoreListScreen

File: components/store/StoreCard.js [hunk 2]
```diff
@@ -57,44 +51,37 @@ export default function StoreCard({
   return (
     <TouchableOpacity
```
Comment (wangannie) [inline components/store/StoreCard.js:52]: This should be conditional on if it's from the `StoreList`. Currently, it seems to be broken bc clicking does nothing but it's navigating to itself

File: components/store/StoreCard.js [hunk 3]
```diff
@@ -57,44 +51,37 @@ export default function StoreCard({
...
-                  ),
-                }}
-                numberOfLines={1}
-                ellipsizeMode="tail">
-                {storeName}
-              </Title>
-            )}
+            <Title
+              style={{
+                maxWidth: getMaxWidth(Dimensions.get('window').width),
```
Comment (wangannie) [inline components/store/StoreCard.js:63]: NICE this works perfectly!

File: lib/mapUtils.js [hunk 4]
```diff
@@ -46,8 +46,8 @@ export async function getProductData(store) {
 // 12px: Extra space for padding between items
 // 130px: Size of 'See Products' button
 // 48px: Size of EBTBar
-export const getMaxWidth = function sync(screenWidth, ebt, seeProducts) {
-  return screenWidth - 32 - 12 + (ebt ? -48 : 0) + (seeProducts ? -130 : 0);
+export const getMaxWidth = function sync(screenWidth) {
+  return screenWidth - 32 - 12 - 30;
```
Comment (wangannie) [inline lib/mapUtils.js:50]: Update comments above to reflect this

File: components/product/ProductCard.js [hunk 5]
```diff
@@ -16,10 +22,17 @@ function ProductCard({ product, store, navigation, displayPoints }) {
...
       }>
       <ColumnContainer>
-        <Image
-          source={{ uri: product.imageUrl }}
-          style={{ height: 86, width: 86, borderRadius: 12 }}
-        />
+        {productsScreen ? (
+          <Image
+            source={{ uri: product.imageUrl }}
+            style={{ height: 86, width: 151, borderRadius: 12 }}
```
Comment (wangannie) [inline components/product/ProductCard.js:28]: AH thank you for doing this! It finally looks normal with the points line!

File: constants/Products.js [hunk 6]
```diff
@@ -0,0 +1,4 @@
+// Number of days between every delivery. Should be represented in word form.
+const deliveryFrequency = 'ten';
```
Comment (wangannie) [inline constants/Products.js:2]: Would maybe make this a number
